axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
+
+    temp = player.history[:]
+    player.history = player.use_history[:]
+
```

Comment:
here's where the state-change is happening.  when the player calls its `original_class.strategy` it still affects all the state in the player.  so, for instance, a `DualFSMPlayer` would still own an FSM, and update its state based on the call to `self.FSMPlayer.strategy`.